hammerspoon/microphone.lua

```diff
@@ -0,0 +1,60 @@
+local message = require('status-message')
+
+local messageMuting = message.new('muted 🎤')
+local messageHot = message.new('hot 🎤')
+local last_mods = {}
+local recently_clicked = false
+local second_click = false
+
+display_status = function ()
```

Comment:
Tiny style-related request: Remove the whitespace between `function` and `(` for consistency with the existing code.